.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: python
 python:
   - 2.7
-  - 3.4
-  - 3.5
   - 3.6
+  - 3.7
```

Comment:
We might want to reflect the support for 3.7/3.8 in our setup.py.

Reply:
Done in 77eba7b6a80ced0a277d604542e5612835337b11